packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,42 +5,54 @@ keywords: Request Client, Payment networks
 
-Request is blockchain agnostic, payment methods that are suppored for automated payment dection are: Bitcoin, Ether and ERC20. If you miss your favorite blockchain, you can either get in touch with the team or **TODO learn how to do it yourself**
+Note that these characteristics are almost the same for every address-based payment network.
 
-## Ethereum and ERC20
+For most payment networks, the payment method can be performed by the payer without any interaction with the Request protocol or the Request client.
+For all payment networks, the Request client can perform the payment detection method to determine the balance of a request.
 
-Because one Ethereum address is generally used many times to receive and send transactions, we need a way to identify each payment. Request provides two payment channels for ETH and ERC20:
+Currencies that are supported for automated payment detection are Bitcoin, Ether, and ERC20. If you miss your favorite blockchain, you can either get in touch with the team or **TODO learn how to do it yourself**
 
-### Address-based payment detection
-The payment channel `pn-erc20-address-based` can be used for one-shot address that will only receives payments for a single request.
+## Bitcoin
 
-Before using this payment channel, you then have to make sure you create two addresses: one for the payment (or payments if the request is paid in several parts), and one for the refund.
+Generating a new address for every inbound BTC transfers is already part of the Bitcoin's good practices. Therefore, Request only has one payment network for bitcoin: `pn-bitcoin-address-based`.
+
+## Ether
+
+Because one Ethereum address is generally used many times to receive and send transactions, we need a way to identify payments for a specific request without having to create a new address. This is why the payment network for ether is based on a reference: `pn-ethereum-input-data`.
+
+The reference is a hash defined by the request id and the payment address (to detect payments) or the refund address (to detect refunds). This reference is either inserted into the Ethereum transaction's input data or documented through a *proxy contract*, a smart contract that forwards ether transfers and stores references.
 
-### Proxy-contract
-This is the most convenient way for most use cases.
+There are two ways to document the reference:
+* Insert the reference in the Ethereum transfer input data when sending ethers to the payment address
+* Call the `transferWithReference` method of the Ethereum proxy smart contract with the payment address and the reference as parameters
 
-If you opt for the `pn-erc20-proxy-contract` payment channel, the flow for your users will be:
+## ERC20
 
-1. The user allows the smart contract to transfer funds from his address, up to a certain amount you choose.
+ERC20 tokens are based on Ethereum and therefore also use Ethereum addresses. Though we provide an address based payment network. 
```

Comment:
"Though we provide an address based payment network. "
We should explain how it works.